packages/vue/src/components/FDatepickerField/update-calendar-value.ts

```diff
@@ -1,26 +1,31 @@
 import { FDate } from "@fkui/date";
 import { isInvalidMonth } from "../../internal-components/calendar/is-invalid-month";
-import FDatepickerField from "./FDatepickerField.vue";
 
 /**
- * @public
+ * @internal
  */
 export function updateCalendarValue(
-    this: InstanceType<typeof FDatepickerField>,
```

Comment:
This broke down in Vue 3.5, it might be related to [`shims-vue.d.ts`](https://github.com/Forsakringskassan/designsystem/blob/main/packages/vue/src/%40types/shims-vue.d.ts) being incompatible but replacing it with small interface solved the issue and made the testcases easier anyway.